ajax/libs/html2pdf.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "html2pdf.js",
```

Comment:
Would you like to use `html2pdf` as `name` which is the same as GitHub repo name?
(As usual, if the library uses git auto-update, we would recommend you use GitHub repo as `name`. But it's not necessary)

Reply:
I changed the GH repo name to match NPM/CDNJS.